net/instaweb/rewriter/rewrite_options.cc

```diff
@@ -995,6 +996,11 @@ std::vector<DeprecatedOptionMap> kDeprecatedOptionNameList(
     kDeprecatedOptionNameData + arraysize(kDeprecatedOptionNameData)
 );
 
+// Will be initialized to a sorted list of headers not allowed in
+// AddResourceHeader.
+const StringPieceVector* fixed_resource_headers_ = NULL;
+bool http_header_separators_[256] = {false};
```

Comment:
Done & pushed. Just to make clear, the line above is ok? (`const StringPieceVector* fixed_resource_headers_`)?